doc/prepare.md

```diff
@@ -0,0 +1,103 @@
+# Prepared Statements
```

Comment:
I feel it's a bit difficult to imagine what the document is from the name `prepare.md` .
Is it better to rename this markdown file with `prepared_statements.md` or something to be more precise?

Reply:
OK! I renamed it.